WebAPI/Endpoints/LibrariesMinimalApi.cs

```diff
@@ -1,5 +1,4 @@
-﻿using Amazon.Util;
-using Application;
+﻿using Application;
```

Comment:
**Suggestion:** The `Amazon.Util` namespace was removed. If this namespace was used in the file, ensure that the removal does not cause any issues. [possible issue]
```suggestion
﻿using Application;
using Amazon.Util;
```